src/qz/printer/status/printer/CupsPrinterStatusMap.java

```diff
@@ -44,11 +46,811 @@
+    WRAPPER_AT_LIMIT(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-at-limit
+    WRAPPER_CLOSED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-closed
+    WRAPPER_CONFIGURATION_CHANGE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-configuration-change
+    WRAPPER_COVER_CLOSED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-cover-closed
+    WRAPPER_COVER_OPEN(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-cover-open
+    WRAPPER_EMPTY(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-empty
+    WRAPPER_FULL(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-full
+    WRAPPER_INTERLOCK_CLOSED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-interlock-closed
+    WRAPPER_INTERLOCK_OPEN(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-interlock-open
+    WRAPPER_JAM(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-jam
+    WRAPPER_LIFE_ALMOST_OVER(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-life-almost-over
+    WRAPPER_LIFE_OVER(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-life-over
+    WRAPPER_MEMORY_EXHAUSTED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-memory-exhausted
+    WRAPPER_MISSING(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-missing
+    WRAPPER_MOTOR_FAILURE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-motor-failure
+    WRAPPER_NEAR_LIMIT(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-near-limit
+    WRAPPER_OFFLINE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-offline
+    WRAPPER_OPENED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-opened
+    WRAPPER_OVER_TEMPERATURE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-over-temperature
+    WRAPPER_POWER_SAVER(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-power-saver
+    WRAPPER_RECOVERABLE_FAILURE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-recoverable-failure
+    WRAPPER_RECOVERABLE_STORAGE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-recoverable-storage
+    WRAPPER_REMOVED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-removed
+    WRAPPER_RESOURCE_ADDED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-resource-added
+    WRAPPER_RESOURCE_REMOVED(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-resource-removed
+    WRAPPER_THERMISTOR_FAILURE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-thermistor-failure
+    WRAPPER_TIMING_FAILURE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-timing-failure
+    WRAPPER_TURNED_OFF(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-turned-off
+    WRAPPER_TURNED_ON(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-turned-on
+    WRAPPER_UNDER_TEMPERATURE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-under-temperature
+    WRAPPER_UNRECOVERABLE_FAILURE(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-unrecoverable-failure
+    WRAPPER_UNRECOVERABLE_STORAGE_ERROR(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-unrecoverable-storage-error
+    WRAPPER_WARMING_UP(NativePrinterStatus.UNKNOWN_STATUS), // wrapper-warming-up
+
+    //printer-state
+
+    IDLE(NativePrinterStatus.OK), // idle
```

Comment:
Does this belong higher? e.g. "CUPS defined states (defined by CUPS, but not part of the IPP specification" ?